Nodejs/Product/Nodejs/Project/NodejsFolderNode.cs

```diff
@@ -100,14 +100,7 @@ class NodejsFolderNode : CommonFolderNode {
 
         public override string Caption {
             get {
-                var res = base.Caption;
-
-                if (NodejsPackage.Instance.GeneralOptionsPage.ShowBrowserAndNodeLabels &&
-                    NodejsPackage.Instance.IntellisenseOptionsPage.AnalysisLevel != AnalysisLevel.Preview &&
-                    _containsNodeOrBrowserFiles) {
-                    res = AppendLabel(res, ContentType);
```

Comment:
You should also be able to remove the `UpdateContentType` method and other related logic, as that only applies to the old analyzer. This should also result in a nice perf boost :smiley: